app/views/posts/index.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1>Posts List</h1>
-<p>list of posts for any user</p>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>See user's posts</title>
+</head>
+<body>
+  <main id ="container"> 
+  <%= render partial:'users/user', locals:{user: @user} %>
+   <% @posts.each do |post| %>
+  <%= render post%>
+   <section class="comment-section">
+    <%= render partial:'comments', as: 'comment', collection:post.recents_comments%>
```

Comment:
- [ ] Great work in fixing the issue raised by the previous reviewer but after creating a comment, I got the following error, kindly fix it.
>![image](https://user-images.githubusercontent.com/53402600/206861300-eb0b7f64-069c-45c4-a914-7b473e563084.png)